atoma-confidential/src/key_management.rs

```diff
@@ -212,6 +212,7 @@ impl X25519KeyPairManager {
     /// * `Err(KeyManagementError)` if the write failed
     pub fn write_private_key_to_file(&self) -> Result<()> {
         use std::fs::{self, create_dir_all};
+        #[cfg(unix)]
```

Comment:
why this ?

Reply:
because I want be able to compile the atoma-node on windows ;-)